src/components/dialog/information/CreditsDialog/CreditsDialogBase.tsx

```diff
@@ -0,0 +1,126 @@
+import {
+  Accordion,
+  AccordionContent,
+  AccordionItem,
+  AccordionTrigger,
+} from "@/components/ui/accordion";
+import { Button } from "@/components/ui/button";
+import {
+  Dialog,
+  DialogContent,
+  DialogDescription,
+  DialogHeader,
+  DialogTitle,
+  DialogTrigger,
+} from "@/components/ui/dialog";
+import { Input } from "@/components/ui/input";
+import { Tabs, TabsContent, TabsList, TabsTrigger } from "@/components/ui/tabs";
+import { useToast } from "@/hooks/use-toast";
+import { UserPlus, Users } from "lucide-react";
+
+export default function CreditsDialogBase() {
+  const { toast } = useToast();
+
+  const handleCopy = () => {
+    navigator.clipboard.writeText("https://shadcn-dialog-snippet.vercel.app");
+    toast({
+      title: "Link Copied",
+      description: "Referral link copied to clipboard!",
+    });
+  };
+
+  return (
+    <Dialog>
+      <DialogTrigger asChild>
+        <Button>Credits information</Button>
```

Comment:
```suggestion
        <Button variant="outline" className="w-fit">Credits information</Button>
```
Button component design is different from Card componenet's button design